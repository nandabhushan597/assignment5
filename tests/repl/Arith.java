4 - 9
  3 * 2
  7 / 2
  7 / 0
  7.0 / 0
  3 * "hi"
  4 + "hi" + 5
  4 + 5 + "hi"
  9 / "hi"
  10 % 3
  2 + 3 * 4
  1.0 * 10 / 3
  10 / 3 * 1.0
  
  
