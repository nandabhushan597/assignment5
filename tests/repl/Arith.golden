Exit code: ExitSuccess
Stdout:
Welcome to the Java interpreter.
Type `:?` at the prompt for the list of commands.
Java> -5
Java> 6
Java> 3
Java> ArithmeticException: / by zero
Java> Infinity
Java> String is not convertible to a numeric type.
Java> @6 => "4hi5"
Java> @10 => "9hi"
Java> String is not convertible to a numeric type.
Java> 1
Java> 14
Java> 3.3333333333333335
Java> 3.0
Java> Java> Java> Good-bye.

Stderr:
